sources/packages/backend/apps/queue-consumers/src/processors/schedulers/student-application-notifications/_tests_/student-application-notifications.scheduler.e2e-spec.ts

```diff
@@ -0,0 +1,7 @@
+describe("StudentApplicationNotificationsScheduler (e2e)", () => {
```

Comment:
Please remove this dummy test file as, this does not have any coverage for the files in the PR